crossandeancommands/make_nexus.py

```diff
@@ -0,0 +1,82 @@
+"""
+Create a nexus file from the most recent data on Edictor.
+"""
+from pyedictor import fetch
+from lingpy import Wordlist
+from lingpy.convert.strings import write_nexus
+
+def run(args):
+
+    with open("./outputs/crossandean.tsv", "w",
+            encoding="utf-8") as f:
+            f.write(fetch("crossandean",
+                    columns=[
+                            "ID", 
+                            "ALIGNMENT", 
+                            "COGIDS", 
+                            "CONCEPT", 
+                            "DOCULECT", 
+                            "FORM", 
+                            "TOKENS", 
+                            "BORROWING", 
+                            "SOURCE", 
+                            "SUBGROUP"
+                            ]
+                    ))
+
+    wl = Wordlist('./outputs/crossandean.tsv')                        
+    
+
+    # These are the blacklists for the nexus file
+    blacklist_concepts = [
+            'star', 
+            'green', 
+            'with', 
+            'small', 
+            'lip', 
+            'wing', 
+            'breast', 
+            'suck, to', 
+            'husband', 
+            'branch'
+            ]
+
+    blacklist_subgroups = [
+            'Aymara', 
+            'Uru-Chipaya'
+            ]
+
+    blacklist_borrowings = [
+            'aymara', 
+            'esp', 
+            'uru'
+            ]
+
+    blacklist_historical = [
+            'CuzquenoAntiguo',
+            'Anonimo', 
+            'SantoTomas',
+            ]
+
+    # This output creates a second wordlist with only the filtered data. 
+    wl.output(
```

Comment:
An easier way is to filter on the fly!

```python
D = {0: [c for c in wl.columns]} # defines the header
for idx in wl:
    if wl[idx, "doculect"] not in blacklist_languages:
        D[idx] = [wl[idx, c] for c in D[0]]
wlnew = Wordlist(D)
```

Reply:
this is implemented with multiple conditions (languages, concepts, borrowings, subgroup) in the most recent commit